apps/callscreen/test/unit/call_screen_test.js

```diff
@@ -807,6 +806,7 @@ suite('call screen', function() {
 
     setup(function() {
       this.sinon.useFakeTimers();
+      this.sinon.spy(navigator.mozL10n, 'setAttributes');
```

Comment:
so, you can spy on setAttributes in your tests, but you can also just check if the data-l10n-id/data-l10n-args is set:

var l10nAttrs = navigator.mozL10n.getAttributes(element);
assert.equal(l10nAttrs.id, 'foo');
assert.deepEqual(l10nAttrs.args, {'n': 5});

I have no preference here, feel free to leave it as is, just offering as an option :)